internal/ir/xds.go

```diff
@@ -818,5 +818,5 @@ type JSONPatchOperation struct {
 	// Refer to https://datatracker.ietf.org/doc/html/rfc6901 for more details.
 	Path string `json:"path"`
 	// Value is the new value of the path location.
-	Value string `json:"value"`
+	Value interface{} `json:"value"`
```

Comment:
try `apiextensionsv1.JSON` ?

Reply:
apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"

Reply:
it worked, thanks !